src/Encamina.Enmarcha.AspNet.OpenApi/Constants.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Encamina.Enmarcha.AspNet.OpenApi;
+
+/// <summary>
+/// Constants used in the OpenAPI library.
+/// </summary>
+public static class Constants
+{
+    /// <summary>
+    /// OpenAPI related constants.
+    /// </summary>
+    public static class OpenApi
+    {
+        /// <summary>
+        /// The OpenAPI key header.
+        /// </summary>
+        public static readonly string KeyHeader = @"x-openapi-key";
```

Comment:
Could it be a constant?

Reply:
That is a good question.
When you have a public constants class with members that could be used from the outside, it is more convenient to have those constants as `static readonly` rather than `const`. The explanation is a bit tricky, so I leave here some sites that explain it better than me 😅

  - https://lassiautio.com/2018/01/21/constant-or-static-readonly/
  - https://next.sonarqube.com/sonarqube/coding_rules?rule_key=csharpsquid%3AS2339&open=csharpsquid%3AS2339